### PR TITLE
[#31] Pass isKeyboardEnabled to EventsBar

### PR DESCRIPTION
**Pull request by dswiecki**
Solves #31

### CODE REVIEW

**Comment by sherubthakur** [comment]
This looks good, I will try to get this merged in a week or so. :)

**Comment by dswiecki** [comment]
@sherubthakur how are things going ? :)

**Comment by sherubthakur** [comment]
@dswiecki I got pulled into some office related work and forgot about this one. Let me add this to me todo list for the weekend :).

File: src/Components/HorizontalTimeline.jsx [hunk 1]
```diff
@@ -75,6 +75,7 @@ class HorizontalTimeline extends React.Component {
         height={props.containerHeight}
         events={events}
         isTouchEnabled={props.isTouchEnabled}
+        isKeyboardEnabled={props.isKeyboardEnabled}
```
Comment (sherubthakur) [inline src/Components/HorizontalTimeline.jsx:78]: Let's add this to the props too.

**Comment by sherubthakur** [comment]
Other than this, I think this looks good. Though my react skills are a bit rusty these days.

**Pull request closed without merging**